src/prop/sat_solver.h

```diff
@@ -170,8 +170,22 @@ class CDCLTSatSolver : public SatSolver
    */
   virtual std::vector<Node> getOrderHeap() const = 0;
 
+  /**
+   * Get proof, which is used if prop-proof-mode is PROOF.
+   * Returns a complete proof computed by this SAT solver.
+   */
   virtual std::shared_ptr<ProofNode> getProof() = 0;
 
+  /**
+   * Get proof, which is used if prop-proof-mode is SKETCH.
```

Comment:
this mode is to be added in follow up PRs?

The semantics is a bit unclear.

Reply:
Yes, it is added in https://github.com/cvc5/cvc5/pull/10324